integration-cli/docker_cli_run_unix_test.go

```diff
@@ -338,3 +338,17 @@ func (s *DockerSuite) TestStopContainerSignal(c *check.C) {
 		c.Fatalf("Expected `exit trapped` in the log, got %v", out)
 	}
 }
+
+func (s *DockerSuite) TestRunSwapLessThanMemoryLimit(c *check.C) {
+	testRequires(c, memoryLimitSupport)
+	testRequires(c, swapMemorySupport)
+	out, exitCode, err := dockerCmdWithError("run", "-m", "16m", "--memory-swap", "15m", "busybox", "echo", "test")
+
+	if err == nil || exitCode == 0 {
```

Comment:
this if block can be replaced by just `c.Assert(err, check.NotNil)` and you can ignore `exitCode` in the command above (just `_` it)